`azurerm_application_gateway` - `ssl_certificate.x.data` must be a base64 string (#21191)
hashicorp/terraform-provider-azurerm: website/docs/r/application_gateway.html.markdown

* `name` - (Required) The Name of the SSL certificate that is unique within this Application Gateway

> * `data` - (Optional) PFX certificate. Required if `key_vault_secret_id` is not set. Must be a base64 encoded string. 

Reviewer: fwiw we describe this as `base64-encoded` in other places, so for consistency:

```suggestion
* `data` - (Optional) The base64-encoded PFX certificate data. Required if `key_vault_secret_id` is not set.
```
